artiq/coredevice/phaser.py

```diff
@@ -549,6 +568,50 @@ def get_dac_temperature(self) -> TInt32:
         """
         return self.dac_read(0x06, div=257) >> 8
 
+    @kernel
+    def dac_sync(self):
+        """Trigger DAC synchronisation for both output channels.
+        If the DAC-NCO is enabled, this applies NCO frequency changes."""
```

Comment:
No. It may do all kinds of things depending on the sync enable bits. And if NCO sync from sif_sync is not set, then it precisely doesn't apply the NCO frequency change.

Reply:
> No. It may do all kinds of things depending on the sync enable bits.

Absolutely. By default, updates to the QMC and NCO frequency are triggered by sif_sync. Of these only the NCO frequency is directly exposed to the user. It seems fair for those using `dac_write` or, changing non-frequency related fields in the `dac` configuration dictionary, should be familiar with the DAC data-sheet.